playwright-e2e/page-objects/local-authority/other-person/laOtherPersonResponsibilityPage.page.ts

```diff
@@ -0,0 +1,21 @@
+import { Locator, Page } from '@playwright/test';
+
+import BasePage from '../../../pages/basePage.page';
+
+export class LAOtherPersonResponsibilityPage extends BasePage {
+  readonly otherPersonResponsibilityHeading: Locator;
+
+  readonly noRadioButton: Locator;
+
+  constructor(page: Page) {
+    super(page);
+
+    this.otherPersonResponsibilityHeading = page.getByRole('heading', { name: 'Is there another person who' });
+
+    this.noRadioButton = page.getByLabel('No', { exact: true });
+  }
+
+  async checkOtherPersonNoRadioButton(): Promise<void> {
+    await this.noRadioButton.check();
```

Comment:
could you reuse the basePage checkNoRadioButton()?

Reply:
Yes you are correct

Reply:
Reference has now been removed, turns out it was not being used